Source/NSButtonCell.m

```diff
@@ -1703,42 +1705,35 @@ - (void) encodeWithCoder: (NSCoder*)aCoder
       [aCoder encodeValueOfObjCType: @encode(BOOL)
               at: &tmp];
 
-      if([NSButtonCell version] <= 2)
```

Comment:
This code was originally put in so that users could save older versions of .gorm files.   This change was very speculative as .gorm file development was not far along at this point and I wanted to provide users a way to create archives for older versions of the libraries.   This hasn't seen much use to my knowledge, so it is okay to remove it as doing so makes the code a lot more straightforward and simple.   Just FYI there is a corresponding change in Gorm which sets the version of certain classes to facilitate this.